ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/DecoderOptions.java

```diff
@@ -79,17 +79,17 @@
    * Return the input {@link DecoderOptions} instance promoted to a {@link DecoderOptions.Parsed}
    * and with all derived fields set.
    */
-  static Parsed parse(DecoderOptions options) {
+  static Parsed parseDecoderOptions(DecoderOptions options) {
     final Parsed parsed = options.as(Parsed.class);
-    enrich(parsed);
+    enrichDecoderOptions(parsed);
     return parsed;
   }
 
   /**
    * Set all the derived fields of a {@link DecoderOptions.Parsed} instance.
    */
-  static void enrich(Parsed options) {
-    SinkOptions.enrich(options);
+  static void enrichDecoderOptions(Parsed options) {
+    SinkOptions.enrichSinkOptions(options);
```

Comment:
I think we can drop the `SinkOptions.` here now?

Reply:
IntelliJ tells me "Static method may be invoked on containing interface class only", so looks like we need to qualify it. Inheritance of static methods seems like a bit of an ugly corner of Java's type system.